components/calendar/src/islamic.rs

```diff
@@ -1465,4 +1463,49 @@ mod test {
             );
         }
     }
+
+    #[ignore]
+    #[test]
+    fn test_days_in_provided_year_observational() {
+        // -1245 1 1 = -214526 (R.D Date)
+        // 1518 1 1 = 764589 (R.D Date)
+        let x_year = -1245;
+        let y_year = 1518;
+
+        let sum_days_in_year: i64 = (x_year..y_year)
+            .map(|year| IslamicObservational::days_in_provided_year(year) as i64)
+            .sum();
+        let expected_number_of_days = IslamicObservational::fixed_from_islamic(IslamicDateInner(
+            ArithmeticDate::new_from_lunar_ordinals(1518, 1, 1).unwrap(),
```

Comment:
Nit, here and below: use `x_year` and `y_year` instead of repeating them.

Reply:
done! I added them as consts.